src/main/java/org/folio/rest/impl/NotificationsResourceImpl.java

```diff
@@ -413,31 +432,36 @@ public void deleteNotifyUserSelf(String olderthan, String lang,
     if (query == null) {
       return; // error has been handled already
     }
-    logger.debug("Deleting self notifications. new query:" + query);
+    logger.debug("deleteNotifyUserSelf:: parameters: olderthan = {}, lang = {}, okapiHeaders = {}",
+      () -> olderthan, () -> lang, () -> headersAsString(okapiHeaders));
     CQLWrapper cql;
     try {
+      logger.debug("deleteNotifyUserSelf:: query = {}", query);
       cql = getCQL(query, -1, -1);
     } catch (Exception e) {
+      logger.warn("deleteNotifyUserSelf:: Deleting _self notifies failed", e);
```

Comment:
how about "notifications" as you did below?

Reply:
done